docker-compose.yml

```diff
@@ -15,6 +15,19 @@ services:
         - .env
       command: "true"
 
+  test-ci:
+      image: app:build
+      ports:
+        - "8000:8000"
+      depends_on:
+        - db
+        - presto
+        - redis
+      env_file:
+        - .env
+      command:
+        "test"
```

Comment:
I am pretty sure this definition isn't needed, we got by fine without it in the previous circle configuration-- you just need to modify how you're calling docker-compose slightly (see the original configuration).

Reply:
The problem is that we're reusing the image from the build step and not mounting the working directory directly anymore. This more accurately tests the contents of the image, but necessitates a different docker-compose command because there isn't a way AFAICT to override (specifically, remove) a volume definition.

Reply:
Ah ok, I don't have any better suggestion so let's just go with this for now!

Reply:
Depending on how docker-compose works, it may be possible to remove https://github.com/mozilla/missioncontrol/blob/master/docker-compose.yml#L12-L13 to get this to work the way you want. It would fix the CI case but if docker-compose doesn't rebuild the container automatically upon invocation it might affect the way you are doing development. Additionally you may be using the functionality of accessing the code directly from the host during container execution, I'm not sure.

For now, I'm merging this, and verifying the deploy step works.